packages/cfd/src/Containers/mt5-create-password/mt5-create-password.scss

```diff
@@ -0,0 +1,249 @@
+            max-height: calc(100vh - 6.4rem);
+        }
+    }
+    @include tablet-up {
+        .cfd-password-modal__content {
+            display: flex;
+            flex-direction: column;
+            width: 100%;
+            margin-bottom: 2rem;
+        }
+    }
+}
+
+.cfd-password-modal {
+    &__warning {
+        padding: 2.5rem;
+        max-width: 50rem;
+        align-self: center;
+        &-text {
+            border-radius: 0.8rem;
+            padding: 0.8rem;
+            background-color: var(--status-warning);
+        }
+    }
+    &__message {
+        max-width: 32rem;
+        margin: auto;
+        line-height: 1.43;
+    }
+    &__create-password {
+        padding: 0 1.2rem;
+        @include mobile-or-tablet-screen {
+            &-body {
+                height: 100%;
+            }
+        }
+
+        &-content {
+            display: flex;
+            flex-direction: column;
+            width: 100%;
+            align-items: center;
+            margin-top: auto;
+            padding-bottom: 1rem;
+            .dc-icon {
+                margin-left: 1.1rem;
+                flex: none;
+            }
+        }
+        &-title {
+            margin-top: 2.4rem;
+            margin-bottom: 0.8rem;
+        }
+        &-description {
+            max-width: 30rem;
+            margin-bottom: 1.6rem;
+        }
+    }
+    &-tnc {
+        margin-block-end: 3.2rem;
+        &--bottom {
+            @include mobile-or-tablet-screen {
+                margin-bottom: 7.5rem;
+            }
+        }
+    }
+    &-info {
+        border-radius: 0.4rem;
+        background: $color-status-information;
+        display: flex;
+        padding: 0.8rem;
+        align-items: flex-start;
+        gap: 0.8rem;
+        margin-block-end: 0.8rem;
+        &__icon {
+            flex-basis: 1.6rem;
+            max-width: 1.6rem;
+        }
+    }
+    &__checkbox {
+        display: flex;
+        align-self: baseline;
+        margin-inline-start: -0.8rem;
+        padding-top: 1rem;
+        .dc-text {
+            font-size: var(--text-size-xxs);
+            text-align: var(--text-align-left);
+            & > a {
+                font-weight: bold;
+                text-transform: lowercase;
+            }
+        }
+    }
+    &-info,
+    &__checkbox {
+        .dc-icon {
+            margin-inline-start: 0;
+        }
+    }
+}
+
+.mt5-password-submit-button {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    align-self: flex-end;
+    background-color: transparent;
+
+    @include mobile-or-tablet-screen {
+        width: 100%;
+    }
+    &:only-child {
+        width: 100%;
+    }
+    .dc-btn {
+        margin: 0 0.8rem;
+
+        &:first-child {
+            margin-left: 0;
+        }
+        &:last-child {
+            margin: 0;
+        }
+
+        @include mobile-or-tablet-screen {
+            width: 100%;
+        }
+    }
+    &--absolute {
```

Comment:
where is this used?

Reply:
it is used in 
![Screenshot 2024-08-26 at 1 18 22 PM](https://github.com/user-attachments/assets/0b6fcdaf-2dd4-4f83-a444-b78de0a3f790)

on testing: when removed it does not load the password modal